<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Graphic</class>
 <widget class="QDialog" name="Graphic">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>650</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>650</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Analyze your function graphically absolutely FREE!</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>791</width>
     <height>651</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="8,1">
    <property name="sizeConstraint">
     <enum>QLayout::SetNoConstraint</enum>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QChartView" name="graphicsView">
      <property name="minimumSize">
       <size>
        <width>600</width>
        <height>600</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>600</width>
        <height>600</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="topMargin">
       <number>40</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <item>
       <widget class="QLabel" name="xmax_label">
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>500</height>
         </size>
        </property>
        <property name="text">
         <string>Xmax</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="xmax_edit">
        <property name="inputMethodHints">
         <set>Qt::ImhDigitsOnly</set>
        </property>
        <property name="text">
         <string>10.0</string>
        </property>
        <property name="maxLength">
         <number>7</number>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="placeholderText">
         <string>X maximum value </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="xmin_label">
        <property name="text">
         <string>Xmin</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="xmin_edit">
        <property name="text">
         <string>-10.0</string>
        </property>
        <property name="maxLength">
         <number>8</number>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="placeholderText">
         <string>X minimum value</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="ymax_label">
        <property name="text">
         <string>Ymax</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="ymax_edit">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>5.0</string>
        </property>
        <property name="maxLength">
         <number>7</number>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="placeholderText">
         <string>Y maximum value </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="ymin_label">
        <property name="text">
         <string>Ymin</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="ymin_edit">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>-5.0</string>
        </property>
        <property name="maxLength">
         <number>8</number>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="placeholderText">
         <string>Y minimum value </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="buttons_label">
        <property name="text">
         <string>Point size</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="horizontalSlider">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="points_per_graph">
        <property name="text">
         <string>Points per graph</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="ppg_slider">
        <property name="minimum">
         <number>600</number>
        </property>
        <property name="maximum">
         <number>1200</number>
        </property>
        <property name="singleStep">
         <number>100</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::TicksBelow</enum>
        </property>
        <property name="tickInterval">
         <number>100</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="autoButton">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Auto</string>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="rebuildButton">
        <property name="text">
         <string>Build graph</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="functionEdit">
        <property name="maxLength">
         <number>256</number>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="placeholderText">
         <string>Edit function </string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>Save image as ...</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QGraphicsView</extends>
   <header>QtCharts</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
